ignite/handlers/checkpoint.py

```diff
@@ -354,7 +358,13 @@ def __call__(self, engine: Engine) -> None:
                 global_step = engine.state.get_event_attrib_value(Events.ITERATION_COMPLETED)
             priority = global_step
 
-        if self._check_lt_n_saved() or self._saved[0].priority < priority:
+        def _compare_fn(old, new) -> bool:
```

Comment:
```suggestion
        @staticmethod
        def _compare_fn(old: float, new: float) -> bool:
```

Reply:
May I know why you want to change this internal function to `static function`? And as it uses `self.greater_or_equal`, users can't call it directly.

Thanks.